restifyOpenapiGenerator.js

```diff
@@ -157,7 +159,6 @@ class RestifyApiGenerate {
             this.parseJoiObject(null, elems[0], data);
         } else {
             const openApiType = joiTypeToOpenApiTypeMap[joiObject.type]; // even if type is object here then ignore and do not go recursive
-            const isRequired = joiObject._flags.presence === 'required';
```

Comment:
Can you explain the reason behind this change?

Reply:
It was only used in the line I removed a little below that, so I cleaned it up :)

Which was removed for this:
> Required isn't a property in schema items, only parameters. It is already set as a seperate array of keys as per the spec

Reply:
I see. Thanks! :D